Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -51,6 +66,37 @@ namespace AWSGameLift
         }
     }
 
+    GameLiftServerProcessDesc AWSGameLiftServerManager::BuildGameLiftServerProcessDesc()
+    {
+        GameLiftServerProcessDesc serverProcessDesc;
+        AZ::IO::FileIOBase* fileIO = AZ::IO::FileIOBase::GetDirectInstance();
+        if (fileIO)
+        {
+            const char pathToLogFolder[] = "@log@/";

```

Comment:
I'm wondering whether it makes sense to allow users to customize the log file paths. Developers may have additional log files they would like to upload like the files under user\eventlogger which are generated by event logger.

Maybe the API can accept an optional list of log file paths to override the default value.

Reply:
good idea, but it should be a separate task